packages/angular/projects/ui-angular/src/index.ts

```diff
@@ -7,6 +7,19 @@ export * from './lib/common/types';
 
 export * from './lib/components';
 export * from './lib/primitives';
-export * from './lib/services';
+export * from './lib/services/authenticator-context.service';
+export * from './lib/services/state-machine.service';
 export * from './lib/directives/amplify-override.directive';
-export * from './lib/proxies';
+export {
+  AmplifyS3Album,
+  AmplifyS3ImagePicker,
+  AmplifyS3Text,
+  AmplifyS3TextPicker,
+  AmplifyS3Image,
+  AmplifyPicker,
+  AmplifyPhotoPicker,
+  AmplifyChatbot,
+  AmplifyToast,
+  AmplifyInput,
+  AmplifyButton,
+} from './lib/proxies';
```

Comment:
Is this the same code that @ErikCH has for #188 ?

Reply:
Yeah, this just makes the exports more explicit because production build was complaining with just `export * from './lib/proxies'`.